Applications/Telemetry/telemetry.py

```diff
@@ -24,10 +25,11 @@
 from flask import request
 from flask_cors import CORS
 
-# Can we clean this up?
-sys.path.append(os.path.join(os.path.dirname(__file__), "../../Faraday_Proxy_Tools/"))  #Append path to common tutorial FaradayIO module
-from FaradayIO import faradaybasicproxyio
-from FaradayIO import telemetryparser
+# Add Faraday library to the Python path.
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
```

Comment:
I modified the import paths and got it to work but I may be removing the intended syntax you wanted @reillyeon with the `faraday.proxyio`:

![image](https://cloud.githubusercontent.com/assets/339144/25066994/d5e33350-21ea-11e7-8603-5c531cb648ab.png)